express_relay/sdk/js/src/examples/simpleSearcherLimo.ts

```diff
@@ -59,13 +60,24 @@ class SimpleSearcherLimo {
     );
   }
 
+  async getMintDecimals(mint: PublicKey): Promise<number> {
+    if (this.mintDecimals[mint.toBase58()]) {
+      return this.mintDecimals[mint.toBase58()];
+    }
+    const decimals = await getMintDecimals(this.connectionSvm, mint);
+    this.mintDecimals[mint.toBase58()] = decimals;
+    return decimals;
+  }
+
   async generateBid(opportunity: OpportunitySvm) {
     const order = opportunity.order;
-    const inputMintDecimals = await this.clientLimo.getOrderInputMintDecimals(
-      order
+    const limoClient = new limo.LimoClient(
```

Comment:
any particular reason to create this here as opposed to once in the constructor? there are no background calls, so shouldn't be wasteful to call it once

Reply:
i guess this is just because the `globalConfig` can technically be different for different orders?